keps/sig-storage/2639-secret-protection/kep.yaml

```diff
@@ -0,0 +1,47 @@
+title: secret protection
+kep-number: 2639
+authors:
+  - "@mkimuram"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-storage
+status: provisional
+creation-date: 2021-04-20
+reviewers:
+  - TBD
+approvers:
+  - TBD
+prr-approvers:
```

Comment:
you don't need PRR approvers in `kep.yaml` anymore, but please create a prod-readiness/sig-storage/2639.yaml file and assign me as PRR approver in there. Thanks!

Reply:
@johnbelamaric 

Fixed as suggested. PTAL
Also, thank you very much for being PRR approver.